### ISSUE TITLE
Some users do not return any tweets

### ISSUE THREAD

**Issue by pluja** [David-Lor/pnytter#8]
I don't really know what can be causing it, I am trying to figure it out. But I have found that some users do not show any tweets when scraped by Pnytter, it just returns an empty array. But on Nitter you can see all of their tweets.

Here is an example account I found, and I am facing the problem: `thomasfred2584`

**Comment by pluja** [comment]
As a side note, I am using your `Pnytter` library as the "unofficial twitter API" for my [iseeyourcash](https://codeberg.org/pluja/iseeyourcash) project :)

**Comment by David-Lor** [comment]
I think that problem may be related with the Nitter instance being used, more specifically with the region. I know about some tweets not being shown because they are censored or legally restricted in some regions, but did not consider that problem for users as well.

On the example you posted, I can find the user on the official Nitter instance (https://nitter.net/thomasfred2584), but not on nitter.domainglass.net (https://nitter.domain.glass/search?f=users&q=thomasfred2584)

**Comment by pluja** [comment]
I see, but either way, the instance I am using (a local one) does show the user and I can see the tweets from the UI. It's a bit strange what happens.

**Comment by pluja** [comment]
It does have tweets, but these are replies to other users:
https://nitter.net/thomasfred2584/with_replies

I'm pretty sure Pnytter also looks at replies because I have seen them from other users. This is why I think it is strange.

**Comment by David-Lor** [comment]
@pluja The problem seems to be that Nitter is no longer capable of correctly searching tweets through the Search feature. I've been trying it on different instances and different users. In some cases tweets are returned but I guess it's because they are cached on the instances, or some weird stuff happened on Twitter's side. Little we can do over here. We may rather ask/keep an eye on nitter's repository.

**Comment by pluja** [comment]
I'm not sure which approach are you using for scrapping a user's tweets, as I haven't digged deep into your code, but from what you say above seems you are using the `search` endpoint? When I did my own scrapper a while ago, I used the `with_replies` endpoint of a profile:

`https://nitter.net/jack/with_replies`

This endpoint will show all tweets and replies from a user, you just need to sift out the retweets (maybe with an option to do it or not). The pagination with `cursor` is the same. As I said, I haven't looked through the code, and maybe you are already using it.
